### PR TITLE
added about component

### PR DESCRIPTION
**Pull request by ReinisZogots**

### CODE REVIEW

File: src/components/atoms/aboutImage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export interface AboutImageTypes {
+  path: string,
+  altTag: string,
+  className: string
+}
+
+const AboutImage = ({ className, path, altTag }: AboutImageTypes) => {
```
Comment (ilze-dom) [inline src/components/atoms/aboutImage/index.tsx:7]: this one is returning
: JSX.Element[]

Try adding this maybe
const AboutImage = ({ className, path, altTag }: AboutImageTypes): JSX.Element[] => {

**Review by ilze-dom — APPROVED** [review]

File: src/components/atoms/aboutImage/index.tsx (hunk 1 above)
Comment (ReinisZogots) [reply src/components/atoms/aboutImage/index.tsx:7]: Changed to AboutImage:React.FC<AboutImageTypes> = ({ className, path, altTag } => {

File: src/components/molecules/aboutImages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Structure from "../../../structure.json";
+import { AboutImageTypes } from "../../atoms/aboutImage";
+import { AboutDataTypes } from "../../organisms/about";
+import AboutImage from "../../atoms/aboutImage";
+
+const AboutImages: React.FC = () => {
+  const AboutData: AboutDataTypes = Structure.about;
+  const AboutImages: AboutImageTypes[] = AboutData.aboutImages;
```
[the remark this answers is not in the document]
Comment (ReinisZogots) [reply src/components/molecules/aboutImages/index.tsx]: changed

File: src/components/molecules/aboutText/index.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { AboutDataTypes } from '../../organisms/about';
+import Structure from "../../../structure.json"
+
+const AboutText:React.FC = () => {
+  const AboutData:AboutDataTypes = Structure.about;
```
[the remark this answers is not in the document]
Comment (ReinisZogots) [reply src/components/molecules/aboutText/index.tsx]: changed

**Review by Jalal999 — APPROVED** [review]

**Pull request merged**